librz/egg/egg.c

```diff
@@ -61,6 +61,9 @@ RZ_API RzEgg *rz_egg_new(void) {
 		goto beach;
 	}
 	egg->remit = &emit_x86;
+	if (egg->remit->init) {
+		egg->remit->init(egg);
+	}
```

Comment:
this should be done elsewhere.

Reply:
also call fini in `rz_egg_setup` to ensure when we change arch, the context is also freed.

Reply:
it's ok in `rz_egg_new`. just add it in `rz_egg_setup`.